### PR TITLE
builder fixes

### PR DESCRIPTION
**Pull request by atolopko-czi**
* slow down API requests to avoid DOS'ing Discover API (temporary fix)
* remove reading of tombstone attribute in API response (no longer exists)
* aws scripts allow parameterization of which disk devices to use: `swap_instance_storage.sh` takes a count, while `mount_instance_storage.sh` takes list of actual devices (can be improved)

### CODE REVIEW

File: tools/cell_census_builder/manifest.py [hunk 1]
```diff
@@ -65,15 +65,15 @@ def load_manifest_from_CxG() -> List[Dataset]:
         }
         for collection in collections
         for dataset in collection["datasets"]
-        if dataset["tombstone"] is False  # ignore anything that has been deleted
```
Comment (atolopko-czi) [inline tools/cell_census_builder/manifest.py:68]: API no longer provides this

File: tools/cell_census_builder/manifest.py [hunk 2]
```diff
@@ -65,15 +65,15 @@ def load_manifest_from_CxG() -> List[Dataset]:
...
     logging.info(f"Found {len(datasets)} datasets, in {len(collections)} collections")
 
     # load per-dataset schema version
-    with concurrent.futures.ThreadPoolExecutor(max_workers=8) as tp:
+    with concurrent.futures.ThreadPoolExecutor(max_workers=4) as tp:
         dataset_metadata = tp.map(
             lambda d: fetch_json(
-                f"{CXG_BASE_URI}curation/v1/collections/{d['collection_id']}/datasets/{d['dataset_id']}"
+                f"{CXG_BASE_URI}curation/v1/collections/{d['collection_id']}/datasets/{d['dataset_id']}",
+                delay_secs=1
```
Comment (atolopko-czi) [inline tools/cell_census_builder/manifest.py:76]: not actually sure if the max_workers or delay was the key to success, but even together this adds insignificant time to the overall build, so not tuning further for now.  Data Portal team has been notified: https://github.com/chanzuckerberg/single-cell-data-portal/issues/3535#issuecomment-1401114636

File: tools/cell_census_builder/scripts/aws/mount_instance_storage.sh [hunk 3]
```diff
@@ -9,6 +9,8 @@
 # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/raid-config.html
 #
 
+DEVICES="$@"
```
Comment (bkmartinjr) [inline tools/cell_census_builder/scripts/aws/mount_instance_storage.sh:12]: super tiny nit - now that it takes params, maybe add a comment at top of file showing usage

File: tools/cell_census_builder/manifest.py (hunk 2 above)
Comment (bkmartinjr) [reply tools/cell_census_builder/manifest.py:76]: Editorializing: that bug in the underlying service really needs to be fixed!

**Review by bkmartinjr — APPROVED** [review]

File: tools/cell_census_builder/scripts/aws/mount_instance_storage.sh (hunk 3 above)
Comment (atolopko-czi) [reply tools/cell_census_builder/scripts/aws/mount_instance_storage.sh:17]: done!

File: tools/cell_census_builder/scripts/aws/swapon_instance_storage.sh [hunk 4]
```diff
@@ -4,6 +4,8 @@
 #
 # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-store-swap-volumes.html
 
+NUM_DEVICES=$1
```
[the remark this answers is not in the document]
Comment (atolopko-czi) [reply tools/cell_census_builder/scripts/aws/swapon_instance_storage.sh:13]: done!

File: tools/cell_census_builder/manifest.py (hunk 2 above)
Comment (atolopko-czi) [reply tools/cell_census_builder/manifest.py:78]: referenced issue in source code

**Pull request merged**